src/main/java/it/gov/pagopa/payhub/activities/dao/FlussoTesoreriaPIIDao.java

```diff
@@ -0,0 +1,7 @@
+package it.gov.pagopa.payhub.activities.dao;
+
+import it.gov.pagopa.payhub.activities.dto.treasury.FlussoTesoreriaPIIDTO;
+
+public interface FlussoTesoreriaPIIDao {
```

Comment:
The PII segregation is not demanded to the activity, but to the DAO!
Activity should not be aware of such persistence requirement

Reply:
deleted FlussoTesoreriaPIIDao and all references